extensions/interactions/interaction_specs.json

```diff
@@ -115,7 +115,14 @@
     "is_linear": false,
     "display_mode": "inline",
     "id": "NumericInput",
-    "customization_arg_specs": [],
+    "customization_arg_specs": [{
+      "description": "Allow only input greater than or equal to zero for Number Input interaction.",
+      "name": "placeholder",
```

Comment:
Why are we calling this customisation arg "placeholder" -- it does not seem to match with what the description says? Maybe see this for example:

https://github.com/oppia/oppia/blob/be8c41fd7858f5eb700002cca003817509ba99f5/extensions/interactions/interaction_specs.json#L29-L33

Reply:
done